doc/developers/maintainer.rst

```diff
@@ -107,34 +113,77 @@ under the ``doc/whats_new/`` folder so PRs that target the next version can
 contribute their changelog entries to this file in parallel to the release
 process.
 
-Minor version release
-~~~~~~~~~~~~~~~~~~~~~
+Minor version release (also known as bug-fix release)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The minor releases should include bug fixes and some relevant documentation
 changes only. Any PR resulting in a behavior change which is not a bug fix
-should be excluded.
+should be excluded. As an example, instructions are given for the `1.2.2` release.
+
+ - If needed, create a commit reordering the changelog section for the release.
+   This must be done via a PR targeting on ``main``.
+
```

Comment:
This is not specific to minor releases. It's by the way already mentioned in the `before a release` section (point 3).

Reply:
This has been addressed by 4bb998a.